services/web/server/tests/unit/with_dbs/01/test_storage.py

```diff
@@ -20,7 +20,7 @@ def storage_server(event_loop, aiohttp_server, app_cfg):
     app = create_safe_application(cfg)
 
     async def _get_locs(request: web.Request):
-        assert not request.has_body
+        assert not request.can_read_body
```

Comment:
not sure I understand this change

Reply:
that is in fact a deprecation of aiohttp...

Reply:
@GitHK , second point int he description above :-)